src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1735,5 +1759,87 @@ private RendezvousAwaitable<int> InternalReceiveResponseHeadersAsync(WinHttpRequ
 
             return state.LifecycleAwaitable;
         }
+
+        internal bool GetCertificateFromCache(IPAddress ipAddress, [MaybeNullWhen(false)] out byte[] rawCertificateBytes)
```

Comment:
```suggestion
        internal bool GetCertificateFromCache(IPAddress ipAddress, [NotNullWhen(true)] out byte[] rawCertificateBytes)
```
Nit: shouldn't matter for usages here, but it's technically "more correct"

Reply:
I guess it's complaining about
```diff
-internal bool GetCertificateFromCache(IPAddress ipAddress, [NotNullWhen(true)] out byte[] rawCertificateBytes)
+internal bool GetCertificateFromCache(IPAddress ipAddress, [NotNullWhen(true)] out byte[]? rawCertificateBytes)
```